### DISCUSSION TITLE
Connection terminated immediately after connecting

### DISCUSSION THREAD

**Discussion by jbcribz** [LizardByte/community#329] [Sunshine]
Hi,

Somehow my Sunshine-Moonlight solution worked fine a couple of months ago. Yesterday I updated to latest Moonlight and Sunshine. Started stream, and it immediately freeze after connecting. Sunshine says "Stream started" then "Stream paused", screen freeze on the laptop for a couple of sek and then says "connection terminated".

Logs just says "device disconnected". No visible errors. Tried connecting from my TV instead of laptop, still the same happens. All devices are on 5ghz net on my Asus Wifi 5 router running Merlin software. Tried both Private and Public network in Windows. Still the same happens.

Any ideas? I don't know what to troubleshoot more, as there aren't any fishy stuff going on in the logs.

**Comment by Melonso** [comment]
I've had the same issue since yesterday after replacing the motherboard and CPU. Two months ago, I also changed the motherboard and CPU, and everything was working as before. I've already reinstalled Sunshine, motherboard drivers, and network card drivers (wifi 5ghz Asus AX3000). Gamestream connects without any issues, and I managed to briefly get Sunshine to cooperate after reinstalling the network card, but then there were lags every minute for several seconds. Then the problem returned, and it looks the same on my end as it does for you. Moonlight connects, I see a black screen or 2-3 frames for a few seconds, hear interrupted sound, and then disconnect from the host.

[a comment by an automated account is not shown]
